### DISCUSSION TITLE
Is it possible to get DATE_MED without year?

### DISCUSSION THREAD

**Discussion by samal-rasmussen** [moment/luxon#1157] [Q&A] [answered]
Looking at:
https://moment.github.io/luxon/index.html#/formatting?id=presets

Is it possible to get DATE_MED without year?

**Comment by benhunter** [comment]
This can be accomplished with [DateTime.toFormat()](https://moment.github.io/luxon/api-docs/index.html#datetimetoformat) like so:

```javascript
DateTime.now().toFormat('LLL dd')
```

**Reply by samal-rasmussen** [reply]
I would have done that if internationalization was not a concern of mine. I should have specified that it is in the original question. But it should also really be considered the default.

So please do NOT do use hand rolled date formatting strings unless you know for a hard fact that you are only serving a single demographic that understands that particular format.

Hand rolled date formatting strings do NOT handle internationalization like the presets like DATE_MED do.

E.g.  some languages do month before day (Oct 14, 1983 ) while others do day before month (14 oct. 1983). In this particular example the difference is probably just a bit unexpected by the reader, but in the end decipherable as the month is fully spelled out. If you were using a number for the month then it would be impossible to be sure you meant the tenth of January or the first of October unless you use an internationalized preset like DATE_SHORT.

**Comment by icambron** [comment, marked as answer]
Yes, you can make your own presets:

```js
const DATE_MED_WITHOUT_YEAR = { month: "short", day: "numeric" };
DateTime.now().toLocaleString(DATE_MED_WITHOUT_YEAR) // => 'Mar 22'
```
